modules/persistent-storage-csi-tp-enable.adoc

```diff
@@ -2,13 +2,17 @@
 //
 // persistent-storage-csi-vsphere.adoc
 // persistent-storage-csi-azure.adoc
+// persistent-storage-csi-azure-file.adoc
 
 ifeval::["{context}" == "persistent-storage-csi-vsphere"]
 :vsphere:
 endif::[]
 ifeval::["{context}" == "persistent-storage-csi-azure"]
 :azure:
 endif::[]
+ifeval::["{context}" == "persistent-storage-csi-azure-file"]
```

Comment:
I believe we're supposed to cap off modules that use ifeval statements at their ends. Ex: https://raw.githubusercontent.com/openshift/openshift-docs/95217444724c2295c5e4198ebdde9ffe15d827b4/modules/ssh-agent-using.adoc